opennms-provision/opennms-provision-api/src/main/java/org/opennms/netmgt/provision/support/DetectorHandlerNettyImpl.java

```diff
@@ -39,115 +37,72 @@
  *
  * @author Seth
  */
-public class DetectorHandlerNettyImpl<Request, Response> extends SimpleChannelHandler {
+public class DetectorHandlerNettyImpl<Request, Response> extends ChannelInboundHandlerAdapter {
     
     private static final Logger LOG = LoggerFactory.getLogger(DetectorHandlerNettyImpl.class);
     
-    //private DetectFuture m_future;
     private AsyncClientConversation<Request, Response> m_conversation;
 
-    /**
-     * <p>setFuture</p>
-     *
-     * @param future a {@link org.opennms.netmgt.provision.DetectFuture} object.
-     * @param <Request> a Request object.
-     * @param <Response> a Response object.
-     */
-    /*
-    public void setFuture(DetectFuture future) {
-        m_future = future;
-    }
-
-    /**
-     * <p>getFuture</p>
-     *
-     * @return a {@link org.opennms.netmgt.provision.DetectFuture} object.
-     * /
-    public DetectFuture getFuture() {
-        return m_future;
-    }
-    */
-
     /** {@inheritDoc} */
     @Override
-    public void channelOpen(ChannelHandlerContext ctx, ChannelStateEvent event) throws Exception {
+    public void channelActive(ChannelHandlerContext ctx) throws Exception {
         LOG.debug("channelOpen()");
         if(!getConversation().hasBanner() && getConversation().getRequest() != null) {
             Object request = getConversation().getRequest();
-            ctx.getChannel().write(request);
+            ctx.channel().write(request);
         }
     }
 
     /** {@inheritDoc} */
     @Override
-    public void channelClosed(ChannelHandlerContext ctx, ChannelStateEvent event) throws Exception {
+    public void channelInactive(ChannelHandlerContext ctx) throws Exception {
         LOG.debug("channelClosed()");
```

Comment:
Change log message to `channelInactive()` ?

Reply:
done.